pybossa/view/projects.py

```diff
@@ -2683,6 +2833,50 @@ def coowners(short_name):
     return handle_content_type(response)
 
 
+@blueprint.route('/<short_name>/summary/coowners', methods=['POST'])
+@login_required
+def coowners_summary(short_name):
+    """Manage coowners of a project."""
```

Comment:
Is there a way to reuse [coowners](https://github.com/bloomberg/pybossa/blob/master/pybossa/view/projects.py#L2641) for this?